include/micm/process/cuda_process_set.hpp

```diff
@@ -57,23 +84,13 @@ namespace micm
     matrix.n_reactions_ = rate_constants[0].size();
     matrix.n_species_ = state_variables[0].size();
 
-    CudaProcessSetParam processSet;
-    processSet.number_of_reactants_ = number_of_reactants_.data();
-    processSet.reactant_ids_ = reactant_ids_.data();
-    processSet.reactant_ids_size_ = reactant_ids_.size();
-    processSet.number_of_products_ = number_of_products_.data();
-    processSet.product_ids_ = product_ids_.data();
-    processSet.product_ids_size_ = product_ids_.size();
-    processSet.yields_ = yields_.data();
-    processSet.yields_size_ = yields_.size();
-
-    std::chrono::nanoseconds kernel_duration = micm::cuda::AddForcingTermsKernelDriver(matrix, processSet);
+    std::chrono::nanoseconds kernel_duration = micm::cuda::AddForcingTermsKernelDriver(matrix, this->devstruct_);
     return kernel_duration;  // time performance of kernel function
   }
 
   template<template<class> class MatrixPolicy, template<class> class SparseMatrixPolicy>
   requires VectorizableDense<MatrixPolicy<double>> && VectorizableSparse<SparseMatrixPolicy<double>>
-  inline std::chrono::nanoseconds CudaProcessSet::AddJacobianTerms(
+  inline std::chrono::nanoseconds CudaProcessSet::FormJacobianMatrix(
```

Comment:
This function overrides `ProcessSet::AddJacobianTerms` [here](https://github.com/NCAR/micm/blob/be23ff66a96e72dd3dc945b83865ef42994bc5bf/include/micm/process/process_set.hpp#L69-L73), so it needs to have the same name

Reply:
Thanks @mattldawson . I have reverted the function name back to `AddJacobianTerms` and it makes more sense if we will consider aerosol chemistry in MICM in the future besides the gas-phase chemistry.

Reply:
And now the verification test for `cuda_process_set` fails. Let me debug more on this.